static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.js

```diff
@@ -6,20 +6,39 @@ import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
 import once from 'lodash/once';
 import { prebid } from 'commercial/modules/prebid/prebid';
 
-export const setupPrebid: () => Promise<void> = once(() => {
+const isGoogleWebPreview: () => boolean = () =>
+    !!(
+        navigator &&
+        navigator.userAgent &&
+        navigator.userAgent.indexOf('Google Web Preview') > -1
+    );
+
+if (!isGoogleWebPreview()) {
+    require('prebid.js/build/dist/prebid'); // eslint-disable-line global-require
```

Comment:
You know I have to ask! Why use `require()` instead of dynamic `import()`?

Reply:
@SiAdcock I was in that discussion! The reason we use require over import is because it works. Import gets translated to something that breaks the running of the commercial module I believe, as this works completely fine on CODE and locally.

Reply:
That's very interesting, we're using dynamic imports elsewhere. I don't want to block this PR, but I'll have a look myself next week.

Reply:
@SiAdcock I have had a look at the other usages of dynamic imports and they all use the WebPack option of [`webpackMode`](https://webpack.js.org/api/module-methods/#magic-comments).

[For example:](https://github.com/guardian/frontend/blob/master/static/src/javascripts/bootstraps/enhanced/main.js#L69)
```javascript
import(/* webpackMode: "eager" */ 'bootstraps/enhanced/common')
```

I'll give that a go to see if making it eager works, as the default is lazy.